### PR TITLE
basics and customization fenced blocks replaced with standard markdown

### PR DESCRIPTION
**Pull request by Jeeppler**
### replaced all github flavored code blocks with fenced kramdown code blocks.
\+ some smaller changes

**Affected files:**
qubes-doc/security/AntiEvilMaid.md : 2 (replaced)
qubes-doc/security/QubesFirewall.md : 28 (replaced)
qubes-doc/security/SecurityGuidelines.md : 12 (replaced)
qubes-doc/security/SecurityPage.md : 2 (replaced)
qubes-doc/releases/1.0/release-notes.md : 4 (replaced)
qubes-doc/reference/QubesService.md : 2 (replaced)
qubes-doc/configuration/Rxvt.md : 4 (replaced)
qubes-doc/configuration/ResizeRootDiskImage.md : 4 (replaced)
qubes-doc/configuration/ExternalDeviceMountPoint.md : 2 (replaced)
qubes-doc/configuration/ZFS.md : 28 (replaced)
qubes-doc/configuration/Mutt.md : 6 (replaced)
qubes-doc/configuration/Postfix.md : 18 (replaced)
qubes-doc/configuration/ConfigFiles.md : 4 (replaced)
qubes-doc/configuration/AssigningDevices.md : 22 (replaced)
qubes-doc/configuration/Fetchmail.md : 6 (replaced)
qubes-doc/configuration/ResizeDiskImage.md : 16 (replaced)
qubes-doc/configuration/NetworkBridgeSupport.md : 16 (replaced)
qubes-doc/configuration/DiskTRIM.md : 6 (replaced)
qubes-doc/configuration/ExternalAudio.md : 6 (replaced)
qubes-doc/troubleshooting/OutOfmemory.md : 10 (replaced)
qubes-doc/troubleshooting/InstallNvidiaDriver.md : 16 (replaced)
qubes-doc/troubleshooting/SonyVaioTinkering.md : 3 (replaced)
qubes-doc/troubleshooting/NvidiaTroubleshooting.md : 12 (replaced)
qubes-doc/developers/SourceCode.md : 4 (replaced)
qubes-doc/developers/debugging/TestBench.md : 6 (replaced)
qubes-doc/developers/debugging/Profiling.md : 12 (replaced)
qubes-doc/developers/debugging/WindowsDebugging.md : 4 (replaced)
qubes-doc/developers/debugging/AutomatedTests.md : 2 (replaced)
qubes-doc/developers/building/DevelopmentWorkflow.md : 28 (replaced)
qubes-doc/developers/building/QubesR3Building.md : 10 (replaced)
qubes-doc/developers/building/InstallationIsoBuilding.md : 21 (replaced)
qubes-doc/developers/building/QubesBuilder.md : 12 (replaced)
qubes-doc/developers/building/KdeDom0.md : 10 (replaced)
qubes-doc/developers/building/BuildingArchlinuxTemplate.md : 34 (replaced)
qubes-doc/developers/building/BuildingNonFedoraTemplate.md : 8 (replaced)
qubes-doc/developers/fundamentals/GUIdocs.md : 4 (replaced)
qubes-doc/developers/services/Dom0SecureUpdates.md : 1 (replaced)
qubes-doc/developers/services/Qrexec.md : 18 (replaced)
qubes-doc/developers/services/Qrexec3.md : 10 (replaced)
qubes-doc/developers/services/Qrexec3Implementation.md : 10 (replaced)
qubes-doc/developers/CodingStyle.md : 10 (replaced)
qubes-doc/managing-os/LinuxHVMTips.md : 6 (replaced)
qubes-doc/managing-os/HvmCreate.md : 30 (replaced)
qubes-doc/managing-os/Templates/FedoraMinimal.md : 10 (replaced)
qubes-doc/managing-os/UninstallingWindowsTools2.md : 2 (replaced)
qubes-doc/managing-os/WindowsAppVms.md : 16 (replaced)
qubes-doc/managing-os/FedoraTemplateUpgrade18.md : 12 (replaced)
qubes-doc/common-tasks/CopyToDomZero.md : 4 (replaced)
qubes-doc/common-tasks/FullScreenMode.md : 4 (replaced)
qubes-doc/common-tasks/SoftwareUpdateDom0.md : 12 (replaced)
qubes-doc/common-tasks/CopyPaste.md : 6 (replaced)
qubes-doc/common-tasks/SoftwareUpdateVM.md : 8 (replaced)
qubes-doc/common-tasks/DisposableVms.md : 6 (replaced)
qubes-doc/common-tasks/BackupEmergencyRestoreV2.md : 16 (replaced)
qubes-doc/installing/UpgradeToR2B2.md : 16 (replaced)
qubes-doc/installing/UpgradeToR2B1.md : 10 (replaced)
qubes-doc/installing/UpgradeToR2B3.md : 12 (replaced)
qubes-doc/installing/UpgradeToR2.md : 6 (replaced)

### CODE REVIEW

File: basics/UserFaq.md [hunk 1]
```diff
@@ -219,11 +219,7 @@ or
 
 1.  Go to the sysfs (`/sys/bus/pci`), find the right device, detach it from the pciback driver and attach back to the original driver. Replace `<BDF>` with your device, for example `00:1c.2`:
 
-    ```
     echo 0000:<BDF> > /sys/bus/pci/drivers/pciback/unbind
     MODALIAS=`cat /sys/bus/pci/devices/0000:<BDF>/modalias`
     MOD=`modprobe -R $MODALIAS | head -n 1`
     echo <BDF> > /sys/bus/pci/drivers/$MOD/bind
```
Comment (ypid) [inline basics/UserFaq.md]: You can try changing that to:

```
{% highlight bash %}
echo 0000:<BDF> > /sys/bus/pci/drivers/pciback/unbind
MODALIAS=`cat /sys/bus/pci/devices/0000:<BDF>/modalias`
MOD=`modprobe -R $MODALIAS | head -n 1`
echo <BDF> > /sys/bus/pci/drivers/$MOD/bind
{% endhighlight %}
```

File: customization/XFCE.md [hunk 2]
```diff
@@ -39,7 +35,6 @@ Things needs/recommended to be done:
 -   create own favorites menu (currently standard XFCE menu isn't modified to use per-VM subsections, which makes it very inconvenient):
     1.  create \~/.config/menus/favorites.menu, example content:
 
-        ```
         <!DOCTYPE Menu PUBLIC "-//freedesktop//DTD Menu 1.0//EN"
```
Comment (ypid) [inline customization/XFCE.md]:
```
{% highlight html %}
```

:wink: Maybe others as well. `pygmentize -L` lists the supported filetypes.

Comment (Jeeppler) [inline customization/XFCE.md]: what is about this kramdown syntax?
~~~ html
code comes here
~~~

Comment (ypid) [inline customization/XFCE.md]:
~~~ html
    code comes here
    ~~~

Works nicely. The only difference is that the `~~~html` syntax adds line numbers …
Currently not supported with

Comment (Jeeppler) [inline customization/XFCE.md]: the https://github.com/QubesOS/qubesos.github.io repository uses kramdown!

Comment (ypid) [inline customization/XFCE.md]: [Right](https://github.com/QubesOS/qubesos.github.io/blob/12dc47618b0bc4e20bf52dc31ff08223a3693486/_config.yml#L23), thanks for the hint.

File: basics/UserFaq.md (hunk 1 above)
Comment (axon-qubes) [inline basics/UserFaq.md]: Also note that, if you're going to do space indenting, you need **eight** spaces if the code block is in a list, like this one. Click [here](https://github.com/Jeeppler/qubes-doc/blob/b65aa75876957201b0b43170cdbd6cf7696d273e/basics/UserFaq.md#i-created-a-usbvm-and-assigned-usb-controllers-to-it-now-the-usbvm-wont-boot) to see that the formatting is still not correct. (The other pages look good, though.)

**Comment by Jeeppler** [comment]
@marmarek do you want to merge it?

**Pull request merged**